x-pack/plugin/otel-data/src/yamlRestTest/resources/rest-api-spec/test/20_logs.tests.yml

```diff
@@ -0,0 +1,33 @@
+---
+setup:
+  - do:
+      cluster.health:
+        wait_for_events: languid
+  - do:
+      cluster.put_component_template:
+        name: logs-otel@custom
+        body:
+          template:
+            settings:
+              index:
+                routing_path: [unit, attributes.*, resource.attributes.*]
+                mode: time_series
```

Comment:
Was this intentional, using TSDS for logs? If so, maybe worth a comment explaining why

Reply:
and if so -- Is it reasonable to default these to be logsdb indices?

Reply:
Seems to be a copy/paste leftover from the metrics tests.